pkg/k8shandler/fluentd.go

```diff
@@ -452,6 +479,9 @@ func newFluentdPodSpec(cluster *logging.ClusterLogging, trustedCABundleCM *v1.Co
 		ReadOnlyRootFilesystem:   utils.GetBool(true),
 		AllowPrivilegeEscalation: utils.GetBool(false),
 	}
+	exporterContainer.SecurityContext = &v1.SecurityContext{
```

Comment:
@syedriko what do we need here to comply with your security changes.  i believe the only requirement is we need to be able to read `/var/log/containers/*`

Reply:
Just reuse the fluentdContainer.SecurityContext

Reply:
@syedriko used same as fluentdContainer.SecurityContext for exporterContainer.SecurityContext.  One test case is failing .. need your review again to resolve it in e2e-gcp test.